metadata-ingestion/src/datahub/ingestion/graph/client.py

```diff
@@ -1065,6 +1067,22 @@ def parse_sql_lineage(
             default_schema=default_schema,
         )
 
+    def create_tag(self, tag_name: str) -> Dict[Any, Any]:
+        graph_query: str = """
+            mutation($tag_detail: CreateTagInput!) {
+                createTag(input: $tag_detail)
+            }
+        """
+
+        variables = {
+            "tag_detail": {"name": tag_name},
+        }
+
+        return self.execute_graphql(
```

Comment:
should return the urn of the newly created tag

Reply:
done